src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/PosBlockContextRule.cs

```diff
@@ -1,45 +0,0 @@
-﻿using System.Threading.Tasks;
-using Stratis.Bitcoin.Consensus.Rules;
-
-namespace Stratis.Bitcoin.Features.Consensus.Rules.CommonRules
-{
-    /// <summary>
-    /// Context checks on a POS block.
-    /// </summary>
-    public class PosBlockContextRule : PartialValidationConsensusRule
-    {
-        /// <inheritdoc />
-        public override Task RunAsync(RuleContext context)
-        {
-            // TODO: fix this validation code
-
-            //// check proof-of-stake
```

Comment:
What a bout this ?`Limited duplicity on stake`

Reply:
If we don't fix this as part of this PR we should at least create an issue to fix it.